src/Equations/RunawayFluid.cpp

```diff
@@ -846,7 +847,7 @@ real_t RunawayFluid::evaluateNeoclassicalConductivityCorrection(len_t ir, real_t
     else if(!collisionLess){
         // qR0 is the safety factor multiplied by R0
         const real_t qR0 =  rGrid->GetVpVol(ir)*rGrid->GetVpVol(ir)*rGrid->GetBTorG(ir)*rGrid->GetFSA_1OverR2(ir)*rGrid->GetFSA_NablaR2OverR2(ir)
-             / (4*M_PI*M_PI*Constants::mu0*unknowns->GetUnknownData(unknowns->GetUnknownID(OptionConstants::UQTY_I_P))[ir]);     
+             / (4*M_PI*M_PI*Constants::mu0*unknowns->GetUnknownData(OptionConstants::UQTY_I_P)[ir]);     
```

Comment:
From what I can tell, this gets called at least ``nr*nMultiples`` times every time the equation system is rebuilt (not counting when used for building the Jacobian). So this is an example of where I think we should prefetch the ID of ``I_p`` and use that when calling ``GetUnknownData()``. As a matter of fact, I changed this on the ``bounceaverage`` branch last night, so could wait until that branch is ready to merge to fix this (but of course merge ``development`` to ``master`` first).